src/main/java/com/minecolonies/coremod/colony/buildings/AbstractBuilding.java

```diff
@@ -1512,7 +496,7 @@ private IRequestSystemBuildingDataStore getDataStore()
         return getDataStore().getOpenRequestsByRequestableType();
     }
 
-    private Map<Integer, Collection<IToken<?>>> getOpenRequestsByCitizen()
+    protected Map<Integer, Collection<IToken<?>>> getOpenRequestsByCitizen()
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Remove usage of generic wildcard type. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1452)